inference-engine/src/inference_engine/ie_compound_blob.cpp

```diff
@@ -199,7 +199,7 @@ TensorDesc verifyI420BlobInput(const Blob::Ptr& y, const Blob::Ptr& u, const Blo
 TensorDesc getBlobTensorDesc(const Blob::Ptr& blob) {
     if (auto nv12 = dynamic_cast<NV12Blob*>(blob.get())) {
         auto yDesc = nv12->y()->getTensorDesc();
-        yDesc.getDims()[1] += 2;
+        yDesc.getDims()[1] += 2;  /* -S- Why 2? */
```

Comment:
I would say the entire logic is strange. Why we should modify shape at all?

Reply:
if you asked me - I do not know that was done before